linalg/batched/gpu_blas.cpp

```diff
@@ -0,0 +1,202 @@
+// Copyright (c) 2010-2024, Lawrence Livermore National Security, LLC. Produced
+// at the Lawrence Livermore National Laboratory. All Rights reserved. See files
+// LICENSE and NOTICE for details. LLNL-CODE-806117.
+//
+// This file is part of the MFEM library. For more information and source code
+// availability visit https://mfem.org.
+//
+// MFEM is free software; you can redistribute it and/or modify it under the
+// terms of the BSD-3 license. We welcome feedback and contributions, see file
+// CONTRIBUTING.md for details.
+
+#include "gpu_blas.hpp"
+#include "../../general/forall.hpp"
+
+#if defined(MFEM_USE_CUDA)
+#define MFEM_cu_or_hip(stub) cu##stub
+#define MFEM_CU_or_HIP(stub) CU##stub
+#elif defined(MFEM_USE_HIP)
+#define MFEM_cu_or_hip(stub) hip##stub
+#define MFEM_CU_or_HIP(stub) HIP##stub
+#endif
+
+#define MFEM_CONCAT(x, y, z) MFEM_CONCAT_(x, y, z)
+#define MFEM_CONCAT_(x, y, z) x ## y ## z
+
+#ifdef MFEM_USE_SINGLE
+#define MFEM_GPUBLAS_PREFIX(stub) MFEM_CONCAT(MFEM_cu_or_hip(blas), S, stub)
+#elif defined(MFEM_USE_DOUBLE)
+#define MFEM_GPUBLAS_PREFIX(stub) MFEM_CONCAT(MFEM_cu_or_hip(blas), D, stub)
+#endif
+
+#define MFEM_BLAS_SUCCESS MFEM_CU_or_HIP(BLAS_STATUS_SUCCESS)
+
+#if defined(MFEM_USE_CUDA)
+#include <cublas_v2.h>
+#endif
```

Comment:
Minor: this included by `gpu_blas.hpp`, so it is not needed here.

Reply:
Thanks, removed in dc13eac9e60e8e8079ac87649ee91d83b11c57e1.